runqemu-rpi.sh

```diff
@@ -0,0 +1,28 @@
+# Shell script to start Raspberry Pi 3 (64-bit) on QEMU
+#!/bin/sh
+
+KERNEL="buildroot/output/images/Image"
+DTB="buildroot/output/images/bcm2710-rpi-3-b.dtb" 
+ROOTFS="buildroot/output/images/rootfs.ext2"
+
+if [ ! -f $KERNEL ]
+then
+	echo "$KERNEL does not exist"
+	exit 1
+fi
+
+if [ ! -f $DTB ]
+then
+	echo "$DTB does not exist"
+	exit 1
+fi
+
+if [ ! -f $ROOTFS ]
+then
+	echo "$ROOTFS does not exist"
+	exit 1
+fi
+
+
+qemu-system-aarch64 -M raspi3 -kernel $KERNEL -dtb $DTB -m 1024 -nographic -append "rw console=ttyAMA0,115200 root=/dev/mmcblk0 fsck.repair=yes rootwait" -device sd-card,drive=mycard -drive if=none,file=$ROOTFS,format=raw,id=mycard
```

Comment:
@lnxblog could you please use line extensions here, similar to https://github.com/cu-ecen-aeld/buildroot-assignments-base/blob/master/runqemu.sh#L7 ?

Reply:
have made the changes in this [commit](https://github.com/cu-ecen-aeld/buildroot-assignments-base/pull/27/commits/c8ed3c3a8826eb4bfb5a46171c66d93fea6c6271)